### PR TITLE
docs(pt-br): Docs Translation to brazilian portugues.

### PR DESCRIPTION
**Pull request by Riverfount**
This is a first version in PT-BR of Dynaconf Documentation.
At the moment was translated index.md and cli.md

### CODE REVIEW

File: docs/pt-br/docs/index.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+## Instalação
+
+### Instalando via [pypi](https://pypi.org/project/dynaconf)
+
+```bash
+pip install dynaconf
+```
+### Initialize Dynaconf on your project
+
+???+ "Usando somento Python"
```
Comment (jonatasoli) [inline docs/pt-br/docs/index.md:38]:
```suggestion
???+ "Usando somente Python"
```

File: docs/pt-br/docs/index.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+    #### Usando somento Python
```
Comment (jonatasoli) [inline docs/pt-br/docs/index.md:40]:
```suggestion
    #### Usando somente Python
```

File: docs/pt-br/docs/index.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    🎉 Dynaconf is configured! read more on https://dynaconf.com
+    ```
+
+    > ℹ️ Você pode escolher `toml|yaml|json|ini|py` em `dynaconf init -f <fileformat>`,  **toml** é o formato padrão e o mais **recomendado** para configurações.
+
+    O comando `init` do Dynaconf cria os seguintes arquivos
+
+    ```plain
+    .
+    ├── config.py       # Where you import your settings object (required)
```
Comment (jonatasoli) [inline docs/pt-br/docs/index.md:72]:
```suggestion
    ├── config.py       # Aonde você importa seu objeto de configuração  (obrigatório)
```

File: docs/pt-br/docs/index.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    ├── .secrets.toml   # Sensitive data like passwords and tokens (optional)
```
Comment (jonatasoli) [inline docs/pt-br/docs/index.md:73]:
```suggestion
    ├── .secrets.toml   # Arquivos com dados sensíveis como senhas e tokens (opcional)
```

File: docs/pt-br/docs/index.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+    └── settings.toml   # Application setttings (optional)
```
Comment (jonatasoli) [inline docs/pt-br/docs/index.md:74]:
```suggestion
    └── settings.toml   # Configuração da Aplicação (opcional)
```

**Review by jonatasoli — CHANGES_REQUESTED** [review]

File: docs/pt-br/docs/cli.md [hunk 6]
```diff
@@ -0,0 +1,243 @@
...
+Uso: dynaconf [OPÇÕES] COMMAND [ARGS]...
+
+  Dynaconf - Command Line Interface
+
+  Documentation: https://dynaconf.com/
+
+Opções:
+  --version            Mostra a versão do dynaconf
+  --docs               Abre a documentação no navegador
+  --banner             Mostra um baner maneiro
```
[the remark this answers is not in the document]
Comment (Riverfount) [reply docs/pt-br/docs/cli.md]: I fixed the errors pointed out.

File: docs/pt-br/docs/cli.md [hunk 7]
```diff
@@ -0,0 +1,243 @@
...
+  -i, --instance TEXT  Define a instância custumizada do LazySettings
```
[the remark this answers is not in the document]
Comment (Riverfount) [reply docs/pt-br/docs/cli.md]: I fixed the errors pointed out.

File: docs/pt-br/docs/cli.md [hunk 8]
```diff
@@ -0,0 +1,243 @@
...
+  --help               Mostra essa mensagem e sai.
+
+Commands:
+  get       Retorna o valor bruto de uma chave de configuração
+  init      Inicia um projeto dynaconf, por padrão cria um settings.toml ...
+  list      Litas todos os valores de configuração definidos pelo usuário se `--all` é passado ...
```
[the remark this answers is not in the document]
Comment (Riverfount) [reply docs/pt-br/docs/cli.md]: I fixed the errors pointed out.

File: docs/pt-br/docs/index.md (hunk 5 above)
Comment (Riverfount) [reply docs/pt-br/docs/index.md]: I fixed the errors pointed out.

File: docs/pt-br/docs/cli.md [hunk 9]
```diff
@@ -0,0 +1,243 @@
+
+
+## configurando instancias
+
+Todo comando (com exceção do init) requer uma Instância que pode ser passada usano o parâmetro `-i` ou por variável de ambiente com `export INSTANCE_FOR_DYNACONF`
```
[the remark this answers is not in the document]
Comment (Riverfount) [reply docs/pt-br/docs/cli.md]: I fixed the errors pointed out.

File: docs/pt-br/docs/index.md (hunk 1 above)
Comment (Riverfount) [reply docs/pt-br/docs/index.md]: I fixed the errors pointed out.

File: docs/pt-br/docs/index.md (hunk 3 above)
Comment (Riverfount) [reply docs/pt-br/docs/index.md]: I fixed the errors pointed out.

File: docs/pt-br/docs/index.md (hunk 4 above)
Comment (Riverfount) [reply docs/pt-br/docs/index.md]: I fixed the errors pointed out.

**Review by jonatasoli — APPROVED** [review]
image

**Comment by jonatasoli** [comment]
I think change Draft status this PR.

**Review by rochacbruno — APPROVED** [review]

**Pull request merged**